vizro-core/examples/visual-vocabulary/pages/distribution.py

```diff
@@ -79,58 +80,63 @@
+            A histogram is useful for showing your audience where specific values are concentrated, identifying the
+            extremes, and spotting any gaps or outliers. It can also help you visualize a rough probability
+            distribution. Ensure that the gaps between columns are minimal to make the 'shape' of your data
+            immediately clear.
         """
         ),
-        vm.Graph(
-            figure=dumbbell(
-                tips.groupby(["day", "sex"]).agg({"tip": "sum"}).reset_index(), y="day", x="tip", color="sex"
-            )
-        ),
-        make_code_clipboard_from_py_file("dotplot.py"),
+        vm.Graph(figure=px.histogram(tips, x="total_bill")),
+        make_code_clipboard_from_py_file("histogram.py"),
     ],
 )
 
-histogram = vm.Page(
-    title="Histogram",
-    path="distribution/histogram",
+dumbbell = vm.Page(
+    title="Dumbbell",
+    path="distribution/dumbbell",
     layout=vm.Layout(grid=PAGE_GRID),
     components=[
         vm.Card(
             text="""
-            #### What is a histogram?
-
-            A histogram organizes numerical data into columns, with the size of each column representing how frequently
-            values fall within specified ranges. It visualizes data across a continuous interval.
+            #### What is a dumbbell chart?
+            A dumbbell chart emphasizes the gap between two categorical groups. Each data point is depicted by a
+            symbol, typically a circle, representing its quantitative value. These symbols are connected by a line,
+            visually indicating the gap between the two points. Categories or groups are displayed along one axis,
+            while quantitative values are plotted along the other.
 
             &nbsp;
 
             #### When should I use it?
-
-            A histogram is useful for showing your audience where specific values are concentrated, identifying the
-            extremes, and spotting any gaps or outliers. It can also help you visualize a rough probability
-            distribution. Ensure that the gaps between columns are minimal to make the 'shape' of your data
-            immediately clear.
+            Dumbbell charts are ideal for illustrating differences or gaps between two points. They are less cluttered
+            than bar charts, making it easier to compare groups. Common uses include comparing groups, such as showing
+            differences in performance metrics across various categories. Colors can be used to emphasize the direction
+            of changes or to distinguish between categories.
         """
         ),
-        vm.Graph(figure=px.histogram(tips, x="total_bill")),
-        make_code_clipboard_from_py_file("histogram.py"),
+        vm.Graph(
+            figure=dumbbell(
+                tips.groupby(["day", "sex"]).agg({"tip": "sum"}).reset_index(), y="day", x="tip", color="sex"
```

Comment:
I think we should keep data-munging to a bare minimum in this dashboard. Currently we don't do any groupby aggregations elsewhere, so ideally we'd have an example here that doesn't need it too.

e.g. maybe this? I didn't try it out so not sure how it looks. The query is made much more complex which I don't like, but overall I think it's better since then a reader doesn't think they need to do groupby in order to use this plot. What matters is they have a dataframe of the expected schema to use the plot. (Possibly we should add more advanced examples in future to explain the sort of data wrangling with groupby that might be needed to achieve that required schema but I would't do that now)
```
gapminder().query("year.isin([1952, 2007]) and continent == 'Americas'"), y="country", x="lifeExp", color="year"
```

Reply:
Hmm.. your example would fit better in the `time` category then 🤔 which I agree, the dumbbell can also be used for that, but I think for the distribution example a categorical column would be better suited (similar to how we do it for the bar charts)

Reply:
I can also just start a new data set from scratch 😄

Reply:
I've replaced the data set with a dataset I created from scratch. I still need to think whether we actually want to use the dumbbell for the `time` dimension as well, in which case your example is perfect. However, I think there are reasons why the dumbbell is not a suggested chart in the FT visual vocabulary for the time dimension, but instead they use the slope chart: https://github.com/Financial-Times/chart-doctor/blob/main/visual-vocabulary/poster.png

But let me think about it, and we might add a version for the time dimension as well. For now I would merge without 👍